### ISSUE TITLE
Makefile Missing Backslash

### ISSUE THREAD

**Issue by eholly1** [rlworkgroup/garage#1904]
In Makefile the _run-nvidia_ target is missing a backslash:
```
run-nvidia: build-nvidia
	xhost +local:docker
	docker run \
		-it \
		--rm \
		--runtime=nvidia \
		-v /tmp/.X11-unix:/tmp/.X11-unix \
		-v $(DATA_PATH)/$(CONTAINER_NAME):/home/$(user)/code/garage/data \
		-e DISPLAY=$(DISPLAY) \
		-e QT_X11_NO_MITSHM=1 \
		-e MJKEY="$$(cat $(MJKEY_PATH))" \
		--name $(CONTAINER_NAME)  <<-------------------------------------- HERE
		${RUN_ARGS} \
		rlworkgroup/garage-nvidia ${RUN_CMD}
```

**Comment by ryanjulian** [comment]
Thank you! We will merge the fix ASAP.

**Comment by gitanshu** [comment]
This was already fixed on master. Opened a PR #1914 to fix it in release-2020.06 branch

**Comment by gitanshu** [comment]
#1914 was merged. Closing this since no action needs to be taken on master
